docs/api/content-tracing.md

```diff
@@ -1,9 +1,9 @@
-# content-tracing
+# content-trace
```

Comment:
Actually `content-tracing` is the right name, the `content-trace` bellow is a typo.